### ISSUE TITLE
Index VIAF and local names from finding aids as keywords

### ISSUE THREAD

**Issue by escowles** [pulibrary/bibdata#2036]
Proposal reviewed and approved by DSSG: https://docs.google.com/document/d/1Bf5zdehIg1WJMHvrl7XhLX5mQe1wBY_BKWnSUsqhpSI/edit

### Re-index required?

- [ ] Full re-index
- [x] Partial re-index
- [ ] No re-indexing required

**Comment by kevinreiss** [comment]
@regineheberlein when you are back let's discuss and update this ticket.

**Comment by sandbergja** [comment]
One blocker is that viaf is not in the list of allowed values for subfield two in name fields.

**Comment by mzelesky** [comment]
I am running a report to find all VIAF and local names in the entire catalog.

**Comment by regineheberlein** [comment]
As an update, SC is currently reviewing their policies regarding the use of VIAF. 
tagging @amycvo

**Comment by regineheberlein** [comment]
@amycvo has reported to the Archival Authorities Working Group that SC have decided to continue using VIAF authorities.

**Comment by kevinreiss** [comment]
@mzelesky when you have a chance can you update the local heading report you posted earlier to include a column to note which ones come from finding aids. That would be useful context to have. @caroldh @ellen-aa I'd hold off on any review of this without that report having some qualification.

**Comment by mzelesky** [comment]
See https://docs.google.com/spreadsheets/d/14vPLJctblrBeg1cotLP8r3_8EFJQiFf0/edit?usp=sharing&ouid=115523013276933762480&rtpof=true&sd=true for a VIAF report.

See https://docs.google.com/spreadsheets/d/1kb-Y1_Jcct0qOCNOA-3T68N6yJ1_bEMA/edit?usp=sharing&ouid=115523013276933762480&rtpof=true&sd=true for a Local report.

**Comment by regineheberlein** [comment]
Yes, I think we still need to do the same thing in the components plugin. I'll look into it.
